mern-ecommerce/controllers/order.js

```diff
@@ -39,7 +39,7 @@ exports.listOrders = (req, res) => {
           error: errorHandler(error),
         });
       }
-      res.json(orders);
+      res.json(ordders);
```

Comment:
"Typo in variable name, should be 'orders' instead of 'ordders'. It's a typo that can cause unexpected behavior."